code/datums/materials/basemats.dm

```diff
@@ -130,6 +140,16 @@
 	strength_modifier = 1.3
 	categories = list(MAT_CATEGORY_ORE = TRUE, MAT_CATEGORY_RIGID = TRUE)
 	sheet_type = /obj/item/stack/sheet/mineral/titanium
+	armor_modifiers = list("melee" = 1.35, "bullet" = 1.3, "laser" = 1.3, "energy" = 1.25, "bomb" = 1.25, "bio" = 1, "rad" = 1, "fire" = 0.7, "acid" = 1)
+
+/datum/material/runite
```

Comment:
shouldnt sheet_type be in this material, with the new sheets you added

Reply:
My bad, thanks.